src/tasks/python/PythonExtensionHelper.ts

```diff
@@ -128,18 +35,19 @@ export namespace PythonExtensionHelper {
             return undefined;
         }
 
-        const debuggerPath = path.join(pyExt.extensionPath, 'pythonFiles', 'lib', 'python');
-        const oldDebugger = path.join(debuggerPath, 'old_ptvsd');
-        const newDebugger = path.join(debuggerPath, 'new_ptvsd');
+        const version = new semver.SemVer(pyExt.packageJSON.version);
+
+        if (version.compare(minPyExtensionVersion) == -1) {
```

Comment:
```suggestion
        if (version.compare(minPyExtensionVersion) < 0) {
```
Or otherwise, `===`.